docs/index.asciidoc

```diff
@@ -988,11 +988,10 @@ do not use full URL here, only paths, e.g. "/sniff/_nodes/http"
   * Value type is <<boolean,boolean>>
   * There is no default value for this setting.
 
-Enable SSL/TLS secured communication to Elasticsearch cluster. Leaving this
-unspecified will use whatever scheme is specified in the URLs listed in 'hosts'.
-If no explicit protocol is specified plain HTTP will be used. If SSL is
-explicitly disabled here the plugin will refuse to start if an HTTPS URL is
-given in 'hosts'
+Enable SSL/TLS secured communication to Elasticsearch cluster.
+Leaving this unspecified will use whatever scheme is specified in the URLs listed in <<plugins-{type}s-{plugin}-hosts>> or extracted from the <<plugins-{type}s-{plugin}-cloud_id>>.
+If no explicit protocol is specified plain HTTP will be used.
+If SSL is explicitly disabled here the plugin will refuse to start if an HTTPS URL is given in 'hosts'
```

Comment:
> If SSL is explicitly disabled here the plugin will refuse to start if an HTTPS URL is given in 'hosts'

This is only true if the api_key is set, otherwise a configuration such as `output { elasticsearch { hosts => ['https://localhost:9300'] ssl => false }}` will not raise an error at startup.